### ISSUE TITLE
EdgeX sample fails when run from behind a proxy

### ISSUE THREAD

**Issue by whbruce** [dlstreamer/pipeline-server#97]
After building successfully the the sample will fail to run with errors such as the following

```
edgex-core-metadata  | level=ERROR ts=2021-12-02T02:11:09.829467932Z app=edgex-core-metadata source=config.go:135 msg="configuration provider is not available"
edgex-core-metadata  | level=ERROR ts=2021-12-02T02:11:10.845472494Z app=edgex-core-metadata source=config.go:135 msg="configuration provider is not available"
```

**Comment by tobiasmo1** [comment]
https://github.com/intel/video-analytics-serving/issues/97

**Comment by whbruce** [comment]
Fixed in v0.7.1.
